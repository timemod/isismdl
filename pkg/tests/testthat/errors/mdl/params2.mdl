ident x = c2;
param c1  3;
param c1  3 coef1 2 3 coef2 *;
param c2 1 2 3 ;
param c3 1 2 q;
param c4;
