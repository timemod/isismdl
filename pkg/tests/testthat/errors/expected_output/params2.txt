Duplicate parameter name c1
Line: 3 of file mdl/params2.mdl
param c1  3 coef1 2 3 coef2 *;
        ^

syntax error, unexpected '*', expecting number or integer
Line: 3 of file mdl/params2.mdl
param c1  3 coef1 2 3 coef2 *;
                             ^

Name c2 already used as exogenous variable name
Line: 4 of file mdl/params2.mdl
param c2 1 2 3 ;
        ^

syntax error, unexpected ';', expecting number or integer
Line: 5 of file mdl/params2.mdl
param c3 1 2 q;
              ^

syntax error, unexpected ';', expecting number or integer
Line: 6 of file mdl/params2.mdl
param c4;
        ^

